src/envoy/http/jwt_auth/jwt_authenticator.cc

```diff
@@ -214,11 +217,20 @@ bool JwtAuthenticator::OkToBypass() {
     return true;
   }
 
+  // Per the spec
+  // http://www.w3.org/TR/cors/#cross-origin-request-with-preflight-0, CORS
+  // pre-flight requests shouldn't include user credentials.
+  if (headers_->Method() &&
```

Comment:
how about bail out earlier, like in line 63?

Reply:
Done.